examples/smart-patrols/test-cases.yaml

```
@@ -0,0 +1,59 @@
+# TODO: generate this _instead_ of "params_fillable.yaml"  and then edit it
+test1:
+  name: Test 1
+  description: "Test case for patrol-events"
+  params:
+    workflow_details:
+      name: "SMART Patrols Dashboard"
+      description: "A dashboard for visualizing patrol trajectories, patrols events, and time density."
+
+    smart_client_name:
+      EcoscopeWorkflowsNamedConnection: "test"
+
+    # Parameters for 'Set Groupers' using task `set_groupers`.
+    groupers:
+      groupers: # (list[ecoscope_workflows.tasks.groupby._groupby.Grouper], FieldInfo(annotation=NoneType, required=True, description='            Index(es) and/or column(s) to group by, along with\n            optional display names and help text.\n            '))
+        - index_name: "month"
+          display_name: "Month"
+          help_text: "The month in which the patrol began or the event was updated."
+          directive: "%B"
```

```suggestion
        - temporal_index: "%B"
          
```

Following #461 I believe this will need to be like this to work.